test/externalTests/gnosis.sh

```diff
@@ -73,55 +73,21 @@ function gnosis_safe_test
     # them for other presets but that's fine - we want same code run for benchmarks to be comparable.
     # TODO: Remove this when Hardhat adjusts heuristics for IR (https://github.com/nomiclabs/hardhat/issues/3365).
     sed -i "s|\(it\)\(('should not allow to call setup on singleton'\)|\1.skip\2|g" test/core/Safe.Setup.spec.ts
-    # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/3365 gets fixed.
     sed -i 's|\(it\)\(("changes the expected storage slot without touching the most important ones"\)|\1.skip\2|g' test/libraries/SignMessageLib.spec.ts
     sed -i "s|\(it\)\(('can be used only via DELEGATECALL opcode'\)|\1.skip\2|g" test/libraries/SignMessageLib.spec.ts
     sed -i 's|\(describe\)\(("Upgrade from Safe 1.1.1"\)|\1.skip\2|g' test/migration/UpgradeFromSafe111.spec.ts
     sed -i 's|\(describe\)\(("Upgrade from Safe 1.2.0"\)|\1.skip\2|g' test/migration/UpgradeFromSafe120.spec.ts
-
-    # TODO: Remove this when Gnosis merges https://github.com/gnosis/safe-contracts/pull/394
-    sed -i "s|\(function isValidSignature(bytes \)calldata\( _data, bytes \)calldata\( _signature)\)|\1memory\2memory\3|g" contracts/handler/CompatibilityFallbackHandler.sol
-
-    # TODO: Remove this when https://github.com/NomicFoundation/hardhat/issues/3365 gets fixed.
     sed -i "s|it\(('should enforce delegatecall'\)|it.skip\1|g" test/accessors/SimulateTxAccessor.spec.ts
     sed -i "s|it\(('can only be called from Safe itself'\)|it.skip\1|g" test/libraries/Migration.spec.ts
     sed -i "s|it\(('should enforce delegatecall to MultiSend'\)|it.skip\1|g" test/libraries/MultiSend.spec.ts
 
-    # Force nested abstract-provider dependencies to be at version 5.6.0. Version 5.7.0 of @ethersproject/abstract-provider
-    # introduced a new field in FeeData, which causes clashes unless all dependency packages of abstract-provider are pegged
-    # to the same version. As we've already had to peg @ethersproject/contracts to 5.6.0 earlier, we are doing so now with
-    # @ethersproject/abstract-provider as well.
-    jq '.overrides."@ethersproject/abstract-provider"="5.6.0" |
-        .overrides."@ethersproject/abstract-signer@5.6.0"
-                  ."@ethersproject/abstract-provider"="5.6.0"' package.json > package.json.tmp
-    mv package.json.tmp package.json
-
     neutralize_package_lock
     neutralize_package_json_hooks
     force_hardhat_compiler_binary "$config_file" "$BINARY_TYPE" "$BINARY_PATH"
     force_hardhat_compiler_settings "$config_file" "$(first_word "$SELECTED_PRESETS")" "$config_var"
-    # npm@8.3.0+ is required for `overrides` support
-    npm install npm@>8.3.0
     npx npm install
     npx npm install hardhat-gas-reporter
```

Comment:
Can you try getting rid of `npx` completely? Both here, and up top in the file.

Reply:
Works locally, removed.

Reply:
Right. I should have removed it since i'm no longer installing it via npm. Thanks.